modules/storage-persistent-storage-pv.adoc

```diff
@@ -152,7 +152,7 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 //|Ceph RBD  | ✅ | ✅ |✅ |  -
 //|CephFS  | ✅ | ✅ | ✅ |  ✅
 |Cinder  | ✅ | ✅ |- |  -
-|Fibre Channel  | ✅ | ✅ |✅ |  -
+|Fibre Channel  | ✅ | ✅ |✅ | ✅^[4]^
```

Comment:
```suggestion
|Fibre Channel  | ✅ | ✅ |✅ | ✅ ^[4]^
```

Reply:
Please consider adding the missing tiny space between ✅ and ^`.

Reply:
Also please make sure in the last "ReadWriteMany" column all the ✅ symbols are placed vertically straight.

Reply:
space is added. alignment is done but I do observe that the row which has ✅ with this ^[4]^ is not getting aligned. This issue was there even before the changes were made.